### DISCUSSION TITLE
Feature Request - Control volume with mousewheel

### DISCUSSION THREAD

**Discussion by pemeve** [File-New-Project/EarTrumpet#811] [Ideas]
Good afternoon,

Apologies if this has been already requested (could not find it using the search function).

I would like to ask if it would be possible to control the volume with the mousewheel if the mouse pointer is located at the taskbar (similarly to what tb-vol-scroll (https://github.com/dvingerh/tb-vol-scroll) and Volume² (https://github.com/irzyxa/Volume2)) do.

Kind regards,

P.

**Comment by davuxcom** [comment]
Yes. Scroll over the EarTrumpet volume icon (don't click it). This has been a feature for several versions now.

**Reply by pemeve** [reply]
Oh my apologies! 

I did not notice it. Is there a way to show the volume bar when scrolling (using the mousewheel) without clicking it?

Thanks!

**Reply by davuxcom** [reply]
No. Someone asked for some kind of UI that pops up for this. It's a good idea, but not a tiny unit of work. It's a shame the tooltip doesn't update as you scroll, because we are instructing Windows that there is an updated tooltip but because it is showing the shell chooses not to use the updated text.

**Reply by pemeve** [reply]
Ohh that's a shame.

Thanks for the answers!
